src/content/docs/en/guides/styling.mdx

```diff
@@ -469,6 +471,77 @@ import "../styles/global.css";
 ---
 ```
 
+#### Legacy Tailwind 3 support
+
+To add (or keep) support for Tailwind 3 to an Astro project, you will need to have (or keep) an older version (`v5.1.5`) of the official Astro Tailwind integration `@astrojs/tailwind` installed. This version of the integration will be excluded when running Astro's command to upgrade all dependencies. However, you may see a message in your terminal with instructions for upgrading to Tailwind 4. You will also need a legacy Tailwind configuration:
+
+<Steps>
+1. Install `v5.1.5` to your project dependencies using your preferred package manager:
+
+   <PackageManagerTabs>
+     <Fragment slot="npm">
+     ```shell
+     npm install @astrojs/tailwind@5.1.5
```

Comment:
Needs to also install `tailwindcss@3` (there's probably a better version to specify here but you get the idea)